.eslintrc

```diff
@@ -1,13 +1,3 @@
 {
-  "extends": "eslint-config-kinvey-platform",
-  "env": {
```

Comment:
Is there any problem in specifying the environment? After all, it's nothing more than just pointing eslint to what already is.

Reply:
https://eslint.org/docs/user-guide/configuring#specifying-environments

Reply:
Does it cause any errors?

Reply:
I don't know about the errors...
My point is that this project is meant to be run in node.js environment and it contains mocha tests. Specifying this `env` object only indicates to eslint the status quo.